pkg/processor/runtime/rpc/abstract.go

```diff
@@ -171,6 +191,25 @@ func (r *AbstractRuntime) Stop() error {
 	return nil
 }
 
+func (r *AbstractRuntime) signal(signal syscall.Signal) error {
+
+	if r.wrapperProcess != nil && r.configuration.ExplicitAckEnabled {
```

Comment:
why binding `r.configuration.ExplicitAckEnabled` to whether it should be `signal` or not? weird coupling. this if can be kept outside and always signal.